R/combined.R

```diff
@@ -85,6 +85,12 @@ spim_combined_load <- function(path, regions = "all") {
   ret$samples <- agg_samples
   ret$data <- agg_data
 
+  ## Store model demography outputs
+  ret$admissions_demography <- lapply(
+    c(sircovid::regions("all"), "uk"), function(x)
+      spimalot:::spim_extract_admissions_by_age_region(ret$samples[[x]]))
+  names(ret$admissions_demography) <- c(sircovid::regions("all"), "uk")
```

Comment:
Is there any reason you need UK here? Because this will almost certainly break tasks where we run England only (i.e. all paper tasks)

Reply:
Also I feel like this could be a function created below, something like `combined_admissions_demography`?

Reply:
A `message()` would be good for this bit too